api/.env

```diff
@@ -1,12 +1,12 @@
 # Application
-APP_PORT=8000
+APP_PORT=3000
 
 # Database
-PGUSER=postgres
-PGPASSWORD=password
-PGHOST=localhost
-PGPORT=54322
-PGDATABASE=swissgeol
+DB_USER=postgres
```

Comment:
Instead of introducing app-specific env variables, I find it convenient to directly use postgres variables. It makes it a breeze to use tools from the postgresql exosystem like psql when interacting with the db.

Reply:
Converted in https://github.com/swissgeol/ngm/pull/827/commits/6f41070f1908aa780de1cc8d452df9cb13a4f956